port/board/afc-bpm/v3_1/payload.h

```diff
@@ -96,6 +96,8 @@ void vTaskPayload( void *pvParameters );
  */
 void payload_init( void );
 
+#define IPMB_UNCONNECTED    0xA2
```

Comment:
It doesn't make much sense to define it here since it is not specific to the board port, please move it to `modules/ipmb.h`. Also, I think a better name would be `IPMB_ADDR_DISCONNECTED`.